Split/Network/Streaming/SyncUpdateWorker.swift

```diff
@@ -146,11 +161,20 @@ class MySegmentsUpdateV2Worker: UpdateWorker<MySegmentsUpdateV2Notification> {
             return
         }
 
-        if keyList.removed.contains(hashedKey) {
+        if keyList.removed.contains(keyHash) {
             remove(segment: segmentName)
             return
         }
     }
+
+    private func handleBounded(encodedKeyMap: String, compressionUtil: CompressionUtil) throws {
+        let keyMap = try payloadDecoder.decodeAsBytes(payload: encodedKeyMap, compressionUtil: compressionUtil)
+        let index = payloadDecoder.computeKeyIndex(hashedKey: keyHash, keyMapLength: keyMap.count)
```

Comment:
if both `computeKeyIndex` and `isKeyInBitmap` methods are always used together, we can consider moving `computeKeyIndex` logic inside `isKeyInBitmap`.

Reply:
`isKeyInBitmap` signature might be like  `func isKeyInBitmap(keyMap: Data, hashedKey: UInt64) -> Bool`

Reply:
yeah, good catch doc! It makes sense :)